### PR TITLE
fix: improved indentation for code samples

### PR DESCRIPTION
**Pull request by scottrippey**
### Description

Fixed indentation in the code blocks on the one-page example.

Fixes # (issue)

#### Type of Change

Please delete options that are not relevant (including this descriptive text).

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Before:

image

### After:

image

### CODE REVIEW

**Comment by ryan-roemer** [comment]
Was this manually edited or with `yarn build-one-page` which is our generation tool?

**Comment by scottrippey** [comment]
Manually edited!  Didn't know it was generated.  So yeah, my changes will probably get overwritten with this approach.
I'll look into fixing the `build-one-page` script instead.

**Comment by ryan-roemer** [comment]
Ah, I remembered -- we already have a bug for it: https://github.com/FormidableLabs/spectacle/issues/913

**Comment by scottrippey** [comment]
Update: fixed the `build-one-page` script to retain proper indentation.

The committed `one-page.html` was generated by the script.

I also did a visual inspection of the "Markdown" slides, since the indentation also changed there, and they're identical before / after:
image

File: scripts/one-page.js
Comment (scottrippey) [inline scripts/one-page.js:62]: FWIW, I added the `######` to ensure we don't accidentally delete a real `<pre>` tag.

**Comment by scottrippey** [comment]
@ryan-roemer I'll wait for an official stamp of approval before merging.  Post-merge, what needs to be done?

File: scripts/one-page.js
Comment (ryan-roemer) [reply scripts/one-page.js:62]: Great Idea! 

It's out there, but I could see `######` potentially happening in a deck. Can we maybe switch to a much more obnoxious token like `SPECTACLE_ONE_PAGE_TEMP_PRE_TOKEN_START` and `SPECTACLE_ONE_PAGE_TEMP_PRE_TOKEN_STOP` or something?

**Review by ryan-roemer — APPROVED** [review]
Looks great! Thanks so much for this fix!!!

One minor refactor, after which I don't need re-review.

**Comment by ryan-roemer** [comment]
@scottrippey Post-merge, we can either do nothing, or do a patch release. This would mean that the `unpkg` version at: https://unpkg.com/spectacle@8.4.0/examples/one-page.html looks better, but it's not a huge deal.

@carlos-kelly what do you think?

**Comment by carlos-kelly** [comment]
@scottrippey @ryan-roemer I think a patch version is great. I can cut one tomorrow.

**Review by carlos-kelly — APPROVED** [review]

File: scripts/one-page.js
Comment (scottrippey) [reply scripts/one-page.js:62]: Absolutely, good idea.

**Pull request merged**
